Index: subversion/libsvn_subr/deprecated.c
===================================================================
--- subversion/libsvn_subr/deprecated.c	(revision 1694418)
+++ subversion/libsvn_subr/deprecated.c	(working copy)
@@ -1497,6 +1497,7 @@
 #endif /* DARWIN */
 
 #if !defined(WIN32)
+#ifdef SVN_HAVE_GPG_AGENT
 void
 svn_auth_get_gpg_agent_simple_provider(svn_auth_provider_object_t **provider,
                                        apr_pool_t *pool)
@@ -1503,6 +1504,7 @@
 {
   svn_auth__get_gpg_agent_simple_provider(provider, pool);
 }
+#endif /* SVN_HAVE_GPG_AGENT */
 #endif /* !WIN32 */
 
 svn_error_t *
